src/browser/extension/options/MiscellaneousGroup.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+export default ({ options, saveOption }) => {
+  const browserName = typeof chrome === 'undefined' ? 'Firefox' : 'Chrome';
```

Comment:
Firefox implemented WebExtension API, which is chrome compatible, so `chrome` object is present there. So probably we should use `navigator.userAgent.indexOf("Firefox") === -1`.